### ISSUE TITLE
Continue Game Option

### ISSUE THREAD

**Issue by Muids** [fedden/poker_ai#112]
When using poker_ai play, I can only get one deal playing against the bot and then if I want to play again we start again fresh with equal number of chips.

It would be nice to have an option to continue the game until the end, where all players keep their chips from the last round.

**Comment by Muids** [comment]
After reading more on Pluribus I see that the AI is only trained for the situation where all players have equal starting number of chips at 10000.

Another possible feature is to have bet sizes and raises being arbitrary amounts. This is a feature Pluribus had as far as I know.

**Comment by big-c-note** [comment]
Hey @Muids ,

Glad you found yourself to the terminal play! Thanks for using.

Yes, unfortunately, each stack size would have its own learned strategy. Not to mention, the combinations of stack sizes of your opponents would be at play here in determining your optimal strategy. 

In order to do this we would need:
1) A robust stack size translation algorithm to reduce the near-continuous number of stack sizes to a more discrete situation
2) Adjust information sets during strategy creation to include abstracted stack sizes of our bot and opponents.

Number 1 could also be used to translate no limit bet sizing, which is what you're referencing in your latest comment.

Although possible, it's a significant overhaul. Something I'd like for us to get to definitely.

So totally agreed with you and something to aspire to.

**Comment by TatyanaAbbes** [comment]
Yes, it is possible to play poker using a bot, but this is considered unethical and against the rules of most online poker sites. A poker bot is a program that uses artificial intelligence (AI) to make decisions at the poker table. Although the bot can make calculations and decisions faster than a human, it lacks the ability to read the behavior of opponents and adjust its play accordingly, which is a critical skill in poker. Using a bot can also give players an unfair advantage over other players, which can be detrimental to the fairness of the game. Personally, I don't play poker for these reasons. I can at most predict sporting events at times. Especially since I now know how to do it after studying the registration instructions at [https://mostbet-ar.net/registration/.](https://mostbet-ar.net/registration/)
